key-script

```diff
@@ -43,7 +40,7 @@ fi
 
 PW="$($cryptkeyscript "$WELCOME_TEXT")"
 
-if check_yubikey_present; then
+if [ "$check_yubikey_present" = 1 ]; then
```

Comment:
Shoudn't it be 

~~~~bash
if [ "$check_yubikey_present" = "1" ]; then
~~~~
?